google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/scaladsl/ForwardProxyHttpsContext.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.bigquery.scaladsl
```

Comment:
Should this class be in the `scaladsl` package? It's a private class that seems useful for both `javadsl` and `scaladsl`. 

Same applies to `ForwardProxyPoolSettings`.